### DISCUSSION TITLE
Show a shape structure in JSON in Razor page

### DISCUSSION THREAD

**Discussion by dodyg** [OrchardCMS/OrchardCore#11727] [Q&A] [answered]
In Liquid we can just use

`{{ Model.Content | console_log }}` or ` {{ Model.Content | json }}`

In Razor pages,

` @await Helper.LiquidToHtmlAsync("{{ Model | json: true }}", Model.Content.ContentItem)` and `    @await Helper.LiquidToHtmlAsync("{{ Model | console_log }}", i.ContentItem)` work.

The problem is that it's not very obvious. I wonder if I am missing something obvious on how to interrogate a shape details in Razor Pages.

**Comment by jptissot** [comment, marked as answer]
I think there is an Orchard helper called `ConsoleLog` that does the same as the liquid `console_log` filter. Maye all you need is to include the project reference for it to work ?

https://github.com/OrchardCMS/OrchardCore/blob/ed4250cd4dc3d6983b536d41da0458e9f20a4311/src/OrchardCore/OrchardCore.ContentManagement.Display/Razor/OrchardRazorHelperExtensions.cs#L30

**Comment by dodyg** [comment]
Never mind. I found `@Helper.ConsoleLog`. Thanks!
